config.json

```diff
@@ -31,6 +32,7 @@
 		},
 		{
 			"bin": "./target/release/statemint",
+			"chain": "statemint-local",
```

Comment:
if it is all the same and we just want 2 chains here, maybe we can do one as statemint and one as statemine?

Reply:
Yeah we could have 2. We'll also need to add a `--chain=statemint` command later on.